signac/contrib/project.py

```diff
@@ -1424,38 +1424,38 @@ def init_project(cls, *, root=None):
         Raises
         ------
         RuntimeError
-            If the project root path already contains a conflicting project
+            If the project path already contains a conflicting project
             configuration.
 
         """
-        if root is None:
-            root = os.getcwd()
+        if path is None:
+            path = os.getcwd()
 
         try:
-            project = cls.get_project(root=root, search=False)
+            project = cls.get_project(path=path, search=False)
         except LookupError:
-            fn_config = _get_project_config_fn(root)
+            fn_config = _get_project_config_fn(path)
             _mkdir_p(os.path.dirname(fn_config))
             config = read_config_file(fn_config)
             config["schema_version"] = SCHEMA_VERSION
             config.write()
-            project = cls.get_project(root=root)
+            project = cls.get_project(path=path)
         return project
 
     @classmethod
-    def get_project(cls, root=None, search=True, **kwargs):
+    def get_project(cls, path=None, search=True, **kwargs):
         r"""Find a project configuration and return the associated project.
 
         Parameters
         ----------
-        root : str
+        path : str
             The starting point to search for a project, defaults to the
             current working directory.
         search : bool
             If True, search for project configurations inside and above
-            the specified root directory, otherwise only return projects
-            with a root directory identical to the specified root argument
-            (Default value = True).
+            the specified directory, otherwise only return projects with
+            a directory whose path is identical to the specified path
+            argument (Default value = True).
```

Comment:
I would like to see the updated signac-docs page on what a project is now. See my other comment about whether a project "has" a path.